result/neighborsv4.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.1.0
+
+info:
+  title: IPv4 Neighbors models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Neighborsv4.States.Request:
+      description: >-
+        The request to retrieve IPv4 Neighbor state (ARP cache entries) for device network interface(s).
```

Comment:
The request to retrieve IPv4 Neighbor state (ARP cache entries) of a network interface(s).